tests/rest/rules_endpoint.go

```diff
@@ -52,7 +52,7 @@ func readGatheringRules(f *frisby.Frisby) Payload {
 // checkResponse checks if the payload returned by server has proper format
 func checkResponse(f *frisby.Frisby, response Payload) {
 	if response.Version != "1.0" {
```

Comment:
Does this condition make sense in the tests?

Reply:
needs to be replaced by using the actual tag

Reply:
there's no actual tag, it is comparing against this sample file here [tests/conditions/rules.json](https://github.com/RedHatInsights/insights-operator-gathering-conditions-service/pull/123/files#diff-cc8f42e6dec07f07b21db1de2ad4397f90cbafc911d34c900419b473ec9a6a02) where I added an arbitrary version